boa/src/builtins/array/tests.rs

```diff
@@ -1544,7 +1544,7 @@ fn get_relative_end() {
 fn array_length_is_not_enumerable() {
     let mut context = Context::new();
```

Comment:
You can remove this `mut`

Reply:
In fact, it seems that the reference that is being passed must be `&mut`, the build was failing. I updated this.